qa/common/gen_qa_reshape_models.py

```diff
@@ -493,6 +518,116 @@ def create_ensemble_modelconfig(
     emu.create_identity_ensemble_modelconfig(
         "reshape", models_dir, model_version, max_batch, dtype,
         input_shapes, input_model_shapes, output_shapes, output_model_shapes)
+
+
+def create_onnx_modelfile(
+        models_dir, model_version, max_batch,
+        dtype, input_shapes, output_shapes):
+
+    # Onnx use string for variable size dimension, and the same string
+    # will be inferred to have same value for the model run.
+    def normalize_variable_shape(shape, increment_index=True):
```

Comment:
It seems this function should go in the test_util.py and be called shape_to_onnx_shape to match what we did for TF